PresentScreenings/Controllers/FilmInfoDialogController.cs

```diff
@@ -144,6 +144,9 @@ void CreateScreeningsScrollView(ref float yCurr)
 
             // Display the screenings.
             GoToScreeningDialog.DisplayScreeningControls(screenings, screeningsView, GoToScreening, ref _currentScreeningControl);
+
+            // Scroll to the selected screening.
+            GoToScreeningDialog.ScrollScreeningToVisible(App.Controller.CurrentScreening, scrollView);
```

Comment:
Did you consider to inherit from `GoToScreeningDialog`?

Reply:
I did.
But I went for a nice and clean bug fix.
Just reaping the low hanging fruit.